app/onboarding/views/account_activated.py

```diff
@@ -0,0 +1,41 @@
+from app.onboarding.base import onboarding_bp
+from enum import Enum
+from flask import redirect, render_template, request, url_for
+
+
+CHROME_EXTENSION_LINK = "https://chrome.google.com/webstore/detail/simpleloginreceive-send-e/dphilobhebphkdjbpfohgikllaljmgbn"
+FIREFOX_EXTENSION_LINK = "https://addons.mozilla.org/firefox/addon/simplelogin/"
+
+
+class Browser(Enum):
+    Firefox = 1
+    Chrome = 2
+    Other = 3
+
+
+def get_browser() -> Browser:
+    user_agent = request.user_agent
```

Comment:
This request.user_agent seems to be deprecated ... https://werkzeug.palletsprojects.com/en/2.0.x/utils/#module-werkzeug.useragents

Reply:
What a way to break things.. Instead of `werkzeug.useragents.UserAgent` its' not `werkzeug.user_agent.UserAgent`...

Reply:
In the version of `werkzeug` that we are using I can still see the `werkzeug.useragents.UserAgent`
Should we upgrade the dependency then? Because we cannot use the new method if we don't do so

Reply:
I don't think we can upgrade it without upgrading flask. I'll do a little test.

Reply:
Upgrading flask is no easy task. Some deps will break and there are packages that haven't been updated in years and don't support `werkzeug` 2. So I think it's safe for now to leave as it is.

Reply:
Thanks for testing it